src/inspector_agent.cc

```diff
@@ -749,12 +749,6 @@ Agent::Agent(Environment* env)
       host_port_(env->inspector_host_port()) {}
 
 Agent::~Agent() {
-  if (start_io_thread_async.data == this) {
-    CHECK(start_io_thread_async_initialized.exchange(false));
-    start_io_thread_async.data = nullptr;
-    // This is global, will never get freed
-    uv_close(reinterpret_cast<uv_handle_t*>(&start_io_thread_async), nullptr);
-  }
 }
```

Comment:
Suggestion: remove entirely, also from inspector_agent.h.

Reply:
I’ve tried but it gets tricky because `Agent` has `std::unique_ptr` members for types that are only forward-declared in the header, and that conflicts with having the implicit default destructor defined from the header too.